x-pack/plugins/siem/server/lib/detection_engine/rules/patch_rules.ts

```diff
@@ -108,7 +108,6 @@ export const patchRules = async ({
       references,
       note,
       version: calculatedVersion,
-      exceptions_list,
```

Comment:
NOTE: Look above and ensure this is removed from `const nextParams = defaults(` above.